sdk/core/src/main/java/com/klaviyo/coresdk/networking/requests/TrackRequest.kt

```diff
@@ -46,6 +52,9 @@ internal class TrackRequest (
         return json.toString()
     }
 
+    /**
+     * Generates a Unix timestamp to store  as the [timestamp] on this object
```

Comment:
Extra space in there.

Reply:
curious when [] brackets are used in documentation, seems like its generally used for class types but looks like here is for a property?

Reply:
Ya classes or properties can be linked to. I don't intend to just use it for classes even though it may be weighted that way. I'd like to provide linking to property mentions where applicable for easy navigation between the doc pages.